backend/src/federation/client/1_1/FederationClient.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLClient, gql } from 'graphql-request'
```

Comment:
Same code as in 1_0, why?

Reply:
This is only the same test as with the multipe graphql endpoint versions, if it is technicaly possible to communicate with a community on api-version 1_1 getting from dht announcement from database.
For sure if we will need a new api version, then there should be a difference between the version 1_0 and 1_1.